utils.py

```diff
@@ -501,6 +501,48 @@ def inverse(self):
         inv = (1./detA)*C_T
         return inv
 
+    def step(self, u_input, t_start = 0, t_end = 100, points = 1000):
+        """
+        Calculate the time domian step response of a mimotf object.
+        
+        Parameters:
+        -----------
+        
+        u_input:    The values of the input variables to the transfer function.
+                    The values can be given as a 1D list or 1D numpy.array 
+                    object.
+        t_start:    The lower bound of the time domain simulation.
+        t_end:      The upper bound of the time domain simulation.
+        points:     The number of iteration points used for the simulation.
+        
+        Returns:
+        --------
+        
+        tspan:      The time values corresponding with the response data points.
+        G_response: A list of arrays, with a length equal to the amount of 
+                    outputs of the transfer function. The arrays contain the 
+                    step response data of each output.
+        """
+        
+        tspan = numpy.linspace(t_start, t_end, points)
+        
+        G_stepdata = [[0 for i in range(self.shape[0])], [0 for i in \
```

Comment:
You don't need this continuation (`\`) - Python allows line breaks in open brackets.

Reply:
Unnecessary continuation is removed